tfe/data_source_oauth_client.go

```diff
@@ -53,7 +53,7 @@ func dataSourceTFEOAuthClientRead(d *schema.ResourceData, meta interface{}) erro
 	case 1:
 		d.Set("oauth_token_id", oc.OAuthTokens[0].ID)
 	default:
-		return fmt.Errorf("Unexpected number of OAuth tokens: %d", len(oc.OAuthTokens))
+		return fmt.Errorf("Unexpected number of OAuth tokens: %d", len(oc.OAuthTokens)) // nolint:golint,errorlint
```

Comment:
Why was `// nolint:golint,errorlint` added in this case?

Reply:
In this case, the error is explicitly an Interger which is best represented by `%d`, hence I disabled the wrapped error warning here.